integrations/Vero/browser.js

```diff
@@ -0,0 +1,155 @@
+    const removeTags = get(tags, "remove");
+    window._veroq.push([
+      "tags",
+      {
+        id: userId,
+        add: addTags,
+        remove: removeTags,
+      },
+    ]);
+  }
+
+  /**
+   * Identify.
+   * Ref - https://developers.getvero.com/?javascript#users-identify
+   *
+   * @param {Identify} identify
+   */
+  identify(rudderElement) {
+    const { message } = rudderElement;
+    const { traits } = message.context || message;
+    const userId = message.userId || message.anonymousId;
+    let payload = traits;
+    if (userId) {
+      payload = { id: userId, ...payload };
+    }
+    window._veroq.push(["user", payload]);
+    const tags = message.context.integerations?.[("vero", "Vero")]?.tags;
+    if (isDefinedAndNotNull(tags)) this.addOrRemoveTags(tags, userId);
+  }
+
+  /**
+   * Track - tracks an event for a specific user
+   * for event `unsubscribe`
+   * Ref - https://developers.getvero.com/?javascript#users-unsubscribe
+   * for event `resubscribe`
+   * Ref - https://developers.getvero.com/?javascript#users-resubscribe
+   * else
+   * Ref - https://developers.getvero.com/?javascript#events-track
+   *
+   * @param {Track} track
+   */
+  track(rudderElement) {
+    logger.debug("=== In Vero track ===");
+
+    const { message } = rudderElement;
+    const { event, properties } = message;
+    if (!event) {
+      logger.error("[Vero]: Event name from track call is missing!!===");
+      return;
+    }
+    const userId = message.userId || message.anonymousId;
+    switch (event.toLowerCase()) {
+      case "unsubscribe":
+        window._veroq.push(["unsubscribe", userId]);
+        break;
+      case "resubscribe":
+        window._veroq.push(["resubscribe", userId]);
+        break;
+      default:
+        window._veroq.push(["track", event, properties]);
+    }
+
+    const tags = message.context.integerations?.[("vero", "Vero")]?.tags;
+    if (isDefinedAndNotNull(tags)) this.addOrRemoveTags(tags, userId);
+  }
+
+  /**
+   * Page.
+   *
+   * @param {Page} page
+   */
+  page(rudderElement) {
+    logger.debug("=== In Vero Page ===");
+    const { message } = rudderElement;
+    let eventName;
+    if (!message.name && !message.category) {
+      eventName = `Viewed Page`;
+    } else if (!message.name && message.category) {
+      eventName = `Viewed ${message.category} Page`;
+    } else if (message.name && !message.category) {
+      eventName = `Viewed ${message.name} Page`;
+    } else {
+      eventName = `Viewed ${message.category} ${message.name} Page`;
+    }
+    rudderElement.message.event = eventName;
+    this.track(rudderElement);
+  }
+
+  /**
+   * Alias.
+   * Ref - https://www.getvero.com/api/http/#users
+   *
+   * @param {Alias} alias
+   */
+  alias(rudderElement) {
+    const { message } = rudderElement;
+    const { userId, previousId } = message;
+    if (!previousId) {
+      logger.debug("===Vero: previousId is required for alias call===");
```

Comment:
We can return from here as well.

Reply:
@sauravlal-rudderstack rather than putting debug log this should be an `error log` along with a `return` statement

Reply:
okay